### PR TITLE
feat: add all dependencies to be fetched from API

### PR DESCRIPTION
**Pull request by 4bg0P**
This PR aims at adding pagination when requesting Dependencies GitHub GraphQL API, as it is not the case in the default tool.

### CODE REVIEW

File: ghascompliance/octokit/dependabot.py [hunk 1]
```diff
@@ -36,40 +36,61 @@
 # https://docs.github.com/en/graphql/reference/objects#repository
 # https://docs.github.com/en/graphql/reference/objects#dependencygraphdependency
 GRAPHQL_DEPENDENCY_INFO = """\
-{
-    repository(owner: "$owner", name: "$repo") {
-        name
-        licenseInfo {
-            name
+query getRepoDeps($id: ID!, $first: Int!, $after: String) {
+  rateLimit {
```
Comment (gpr) [inline ghascompliance/octokit/dependabot.py:40]: Not sure that you need to get these info

File: ghascompliance/octokit/dependabot.py [hunk 2]
```diff
@@ -36,40 +36,61 @@
 # https://docs.github.com/en/graphql/reference/objects#repository
 # https://docs.github.com/en/graphql/reference/objects#dependencygraphdependency
 GRAPHQL_DEPENDENCY_INFO = """\
-{
-    repository(owner: "$owner", name: "$repo") {
-        name
-        licenseInfo {
-            name
+query getRepoDeps($id: ID!, $first: Int!, $after: String) {
```
Comment (gpr) [inline ghascompliance/octokit/dependabot.py:39]: I'm wondering if we need to have the page size as argument, let's put the max by default (100)

File: ghascompliance/octokit/dependabot.py [hunk 3]
```diff
@@ -36,40 +36,61 @@
...
+    cost
+    remaining
+    resetAt
+  }
+  node(id: $id) {
+    ... on Repository {
+      name
+      isArchived
+      dependencyGraphManifests {
+        pageInfo {
```
Comment (gpr) [inline ghascompliance/octokit/dependabot.py:51]: we don't need pageInfo for the manifests there should never more that 100

**Review by gpr — APPROVED** [review]

File: ghascompliance/octokit/dependabot.py (hunk 3 above)
Comment (4bg0P) [reply ghascompliance/octokit/dependabot.py:45]: I think we need the pageInfo to get hasNextPage and endCusor values inside, and iterate on them.

**Review by melvin-mohadeb — APPROVED** [review]

**Pull request merged**